googleapiclient/model.py

```diff
@@ -273,13 +273,19 @@ def serialize(self, body_value):
         return json.dumps(body_value)
 
     def deserialize(self, content):
+        body = {}
         try:
             content = content.decode("utf-8")
         except AttributeError:
             pass
-        body = json.loads(content)
-        if self._data_wrapper and isinstance(body, dict) and "data" in body:
-            body = body["data"]
+        try:
+            body = json.loads(content)
+        except json.decoder.JSONDecodeError:
+            if isinstance(content, str):
+                body = content
+        else:
+            if self._data_wrapper and isinstance(body, dict) and "data" in body:
+                body = body["data"]
```

Comment:
We can rely on `json.loads(content)` doing the Right Thing here, soe we dont need the type check:
```suggestion
        else:
            if self._data_wrapper and "data" in body:
                body = body["data"]
```